kinetic/distribution.yaml

```diff
@@ -1568,6 +1568,15 @@ repositories:
         release: release/kinetic/{package}/{version}
       url: https://github.com/delftrobotics/dr_base-release.git
       version: 1.0.0-0
+  dyn_tune:
+    doc:
+      type: git
+      url: https://github.com/mehdish89/dyn_tune
+      version: master
+    source:
+      type: git
+      url: https://github.com/mehdish89/dyn_tune
```

Comment:
@mehdish89 the referenced urls must use the full url (in the case of git have the `.git` in the end). You can refer to the travis job output for more details on why the checks fails